exonum/src/proto/schema/exonum/runtime.proto

```diff
@@ -146,9 +146,8 @@ message ArtifactState {
 message InstanceState {
   // Status of a service instance.
   enum Status {
-    // The service instance has no status, i.e. this value corresponds to
-    // `Option::None` on the Rust code level and other corresponds to
-    // `Some(...)`.
+    // No value, e.g. there are no pending changes to the instance
```

Comment:
But with this phrase, I also suggest using `Option` in Rust.

Reply:
Who is the recipient of such suggestion? Does the user of protofiles need to know what Rust type to use?

Reply:
Not only Rust language has optional values. Moreover, this remark is a hint for how to interpret `Status::NONE`.